src/newrelic/alerts.js

```diff
@@ -217,15 +250,17 @@ async function updatePolicy(auth, policy, groupPolicy, monitorId, channelId, pol
     // add notification channel
     console.log('Linking notification channel to alert policy', policy.name);
     try {
-      await request.put('https://api.newrelic.com/v2/alerts_policy_channels.json', {
+      const resp = await fetch('https://api.newrelic.com/v2/alerts_policy_channels.json', {
+        method: 'PUT',
         headers: {
           'X-Api-Key': auth,
         },
-        form: {
-          channel_ids: channelId,
-          policy_id: policy.id,
-        },
+        body: `channel_ids=${encodeURIComponent(channelId)}&policy_id=${encodeURIComponent(policy.id)}`,
```

Comment:
If anyone knows a more elegant way to submit url encoded form data, let me know 😄

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

Reply:
Yup, `URLSearchParams`. You also need to set the `content-type: application/x-www-form-urlencoded`header.

The Fetch API allows  both `URLSearchParams` and `FormData` as body: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#Body

`helix-fetch` doesn't support it yet. Feel free to create a feature request issue.

Reply:
or use https://nodejs.org/docs/latest-v12.x/api/querystring.html